apps/site/test/site_web/integration/support_form_test.exs

```diff
@@ -3,117 +3,114 @@ defmodule CustomerSupportTest do
   Integration tests for the customer support form
   """
   use SiteWeb.IntegrationCase
-  import Wallaby.Query
+  alias Wallaby.{Browser, Element, Query}
 
-  @submit_button css("#support-submit")
-  @photo_input css("#photo", visible: false)
+  @submit_button Query.css("#support-submit")
+  @photo_input Query.css("#photo", visible: false)
+  @no_response_checkbox_label Query.css("#no_request_response_label")
 
   def get_static_test_path(filename) do
     :site
     |> Application.app_dir("priv")
     |> Path.join("test/" <> filename)
   end
 
+  def click_query_result(session, query) do
+    Browser.find(session, query, &Element.click(&1))
+  end
+
+  def submit_form(session), do: click_query_result(session, @submit_button)
+
   describe "customer support form" do
     @tag :wallaby
     test "Shows an error for missing type and comments", %{session: session} do
       session
-      |> visit("/customer-support")
-      |> assert_has(@submit_button)
-      |> click(@submit_button)
-      |> assert_has(css(".form-group.has-danger #comments"))
-      |> assert_has(css(".form-group.has-danger #service"))
+      |> Browser.visit("/customer-support")
```

Comment:
I'm not very familiar with Wallaby, what is the difference between calling `visit` and `Browser.visit`?